docs/3.0.0-beta.x/guides/deployment.md

```diff
@@ -1622,3 +1622,413 @@ heroku open
 Like with project updates on Heroku, the file system doesn't support local uploading of files as they will be wiped when Heroku "Cycles" the dyno. This type of file system is called [ephemeral](https://devcenter.heroku.com/articles/dynos#ephemeral-filesystem), which means the file system only lasts until the dyno is restarted (with Heroku this happens any time you redeploy or during their regular restart which can happen every few hours or every day).
 
 Due to Heroku's filesystem you will need to use an upload provider such as AWS S3, Cloudinary, or Rackspace. You can view the documentation for installing providers [here](../plugins/upload.md#install-providers) and you can see a list of providers from both Strapi and the community on [npmjs.com](https://www.npmjs.com/search?q=strapi-provider-upload-&page=0&perPage=20).
+
+## Google App Engine
+
+In this guide we are going to:
+
+- Create a new Strapi project
+- Configure PostgreSQL for the production enviroment
+- Deploy the app to Google App Engine
+- Add the [Google Cloud Storage file uploading plugin](https://github.com/Lith/strapi-provider-upload-google-cloud-storage) by [@Lith](https://github.com/Lith)
+
+### New Strapi project
+
+:::: tabs
+
+::: tab yarn
+
+Use **yarn** to install the Strapi project (**recommended**). [Install yarn with these docs](https://yarnpkg.com/lang/en/docs/install/)
+
+```bash
+yarn create strapi-app my-project --quickstart
+```
+
+:::
+
+::: tab npx
+
+Use **npm/npx** to install the Strapi project
+
+```bash
+npx create-strapi-app my-project --quickstart
+```
+
+:::
+
+::::
+
+When the setup completes, register an admin user using the form which opens in the browser. This user will be only relevant in local development.
+
+The `sqlite` database is created at `.tmp/data.db`.
+
+Login, but don't add content types yet. Close the browser. Quit the running app.
+
+### Initial commit
+
+This may be a good point to commit the files in their initial state.
+
+``` bash
+cd my-project
+git init
+git add .
+git commit -m first
+```
+
+### Install the Cloud SDK CLI tool
+
+[Cloud SDK: Command Line Interface](https://cloud.google.com/sdk/)
+
+### New App Engine project
+
+Create a new [App Engine](https://console.cloud.google.com/appengine/) project.
+
+Select the region, such as `europe-west`.
+
+Language: Node JS
+
+Environment: Standard
+
+The Flexible enviroment is also available, but incurs more cost.
```

Comment:
Be careful with this information, because _standard_ environment can be more expensive than _flexible_ environment in some case. 
I suggest to specify than _standard_ environment is sufficient for development but not for production.
In my case, flex environment is lower expensive, I've make an experience during one week to compare cost between this two configuration and the cost was multiplicated by 3.

I suggest to purpose two examples of configuration (one in standard environment, second for flex environment).

Reply:
Thank for the heads-up!

I added a note and tabs for the two `app.yaml` examples. As I understand, the config difference is only in either specifying `instance_class: F2` or `env: flex`.

Reply:
Great :) I've read your full documentation and that's seem to be great for me.
I used almost the same configuration but with MySQL. 
Good job 👍 (I was no time to do it ^^)